python/ray/serve/examples/doc/conda_env.py

```diff
@@ -0,0 +1,21 @@
+import requests
+import ray
+from ray import serve
+from ray.serve.config import CondaEnv
+import tensorflow as tf
+
+ray.init()
+client = serve.start()
+
+
+def tf_version(request):
+    return ("Tensorflow " + tf.__version__)
+
+
+client.create_backend("tf1", tf_version, env=CondaEnv(name="ray-tf1"))
```

Comment:
Can we pass in the env name without kwarg? e.g., `CondaEnv("ray-tf1")`

Reply:
Yeah, but then we wouldn't be able to make it a Pydantic model.  I'm fine with that, we can always make it a Pydantic model later if it becomes more complicated

Reply:
Good news, we can use `from pydantic.dataclasses import dataclass` to get a positional argument together with automatic type validation.